CIP-XXXX/README.md

```diff
@@ -0,0 +1,371 @@
+use one of the pre-defined numeric identifiers below, represented by an unsigned integer to minimize ambiguity and
+errors and allow for ease of future expansion based on community need.
+
+| ID  | Name         | Explanation                                  |
+|-----|--------------|----------------------------------------------|
+| 0   | Public Sale  | Funds raised during a public token sale      |
+| 1   | Private Sale | Funds raised during a private token sale     |
+| 2   | ITO/ICO Sale | Funds raised during an initial coin offering |
+
+**TODO**
+
+- [X] Expand on explanation of fields and what purpose they aim to serve.
+- [ ] Add JSON schema docs
+- [ ] Add examples
+
+## Rationale: how does this CIP achieve its goals?
+
+This specification is written and designed to work in conjunction with a registration and validation
+specification such as [CIP-88](https://github.com/cardano-foundation/cips/pull/467). Validation and connection
+of the data contained herein to a specific token project is considered out of scope for the purpose of this
+CIP which should focus on the development and iteration of the disclosure standard.
+
+For the first draft of this CIP the specification has been attempted to model after the CoinMarketCap and
+CoinGecko information requested from their Google Docs solutions<sup>[[1](#fn1),[2](#fn2)]</sup> to attempt to be as
+compatible as possible with existing ecosystem-agnostic platforms for simplicity of integration.
+
+In order to maximize cross-platform compatibility it is recommended that these financial disclosure documents
+be published as well-formed JSON documents utilizing decentralized, quasi-permanent storage solutions such as
+Arweave or IPFS. However, platforms should accept documents published via traditional means such as HTTPS or FTP.
+
+In order to prove the validity of the document, when publishing to the blockchain the document should be
+published in the format of a URI that points to the document itself along with a SHA256 checksum of the document
+contents (similar to how stake pool metadata is currently handled).
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] Complete all "TODO" tasks based on currently identified shortcomings as well as feedback received.
+- [ ] Receive community feedback and refinement from affected parties and potential implementors.
+- [ ] Achieve buy-in from ecosystem participants to analyze and display data based on this spec
+
+### Implementation Plan
+
+- [ ] Invite identified stakeholders and market experts to review and provide feedback to the specification.
+- [ ] Published OSS solutions for generation of FUDS documentation for projects
+- [ ] Published OSS solutions for indexing of FUDS documentation for aggregators
+
+## Copyright
+
+This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
+
+## Footnotes
```

Comment:
Since we don't have a section like this (e.g. I had to eliminate it from my own CIP when standardising it: https://github.com/cardano-foundation/CIPs/pull/429), and given this is hypertext after all, can you fold these footnote links into SEO anchors in the document itself so this section can be eliminated?

If this is not considered necessary then maybe @KtorZ will overrule this suggestion.

Reply:
Yeah, I'm conflicted. I feel like re-introducing the full link and comment about "what it is" where these things are referenced in the context of the CIP distracts from the language of what we're attempting to explain in the CIP... This is why I opted for a Wiki-like "References" section.